.github/workflows/pull.yml

```diff
@@ -513,6 +513,32 @@ jobs:
         # Run pytest without simulator
         backends/arm/test/test_arm_baremetal.sh test_pytest
 
+  unittest-nxp-neutron:
+    uses: pytorch/test-infra/.github/workflows/linux_job_v2.yml@main
+    permissions:
+      id-token: write
+      contents: read
+    with:
+      runner: linux.2xlarge
+      docker-image: executorch-ubuntu-22.04-clang12
+      submodules: 'true'
+      ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
+      timeout: 90
+      script: |
+        set -eux
+        
+        # The generic Linux job chooses to use base env, not the one setup by the image
+        CONDA_ENV=$(conda env list --json | jq -r ".envs | .[-1]")
+        conda activate "${CONDA_ENV}"
+        
+        # Build and install Executorch
+        PYTHON_EXECUTABLE=python \
+        CMAKE_ARGS="-DEXECUTORCH_BUILD_PYBIND=ON -DEXECUTORCH_BUILD_NXP_NEUTRON=ON" \
```

Comment:
Curious, why do we need to enable `PYBIND`?

Reply:
Sorry, this is my misunderstanding of how `EXECUTORCH_BUILD_PYBIND` should be used. I have considered it as a "mechanism to build backend-specific dependencies", because I needed to build `portable_lib` and also `libquantized_ops_aot_lib`. Removed.